roles/kafka/broker/tasks/config.yml

```diff
@@ -16,22 +17,34 @@
   template:
     src: log4j.properties.j2
     dest: "{{ kafka_broker_conf_dir }}/log4j.properties"
+    owner: root
+    group: root
+    mode: "644"
 
 - name: Template server.properties
   template:
     src: server.properties.j2
     dest: "{{ kafka_broker_conf_dir }}/server.properties"
+    owner: root
+    group: root
+    mode: "644"
   vars:
     broker_id: "{{ groups['all'].index(inventory_hostname) }}"
 
 - name: Template kafka-env.conf
   template:
     src: kafka-env.conf.j2
     dest: "{{ kafka_broker_conf_dir }}/kafka-env.conf"
+    owner: root
+    group: root
+    mode: "644"
   vars:
     broker_id: "{{ groups['all'].index(inventory_hostname) }}"
 
 - name: Render jmxremote.password
   template:
     src: jmxremote.password.j2
     dest: "{{ kafka_broker_conf_dir }}/jmxremote.password"
+    owner: root
+    group: root
+    mode: "640"
```

Comment:
Can Kafka read this file ?

Reply:
No and I realized I actually forgot to fix some actions as well for common and ranger. I will mark as draft and reopen once I fix everything.

Reply:
It's still `owner: root, group: root, mode: 640`, which means kafka user cannot read this